providers/aws/resources/aws_vpc.go

```diff
@@ -140,6 +140,10 @@ func (a *mqlAwsVpc) flowLogs() ([]interface{}, error) {
 	return flowLogs, nil
 }
 
+func (a *mqlAwsVpcRoutetable) id() (string, error) {
+  return a.Id.Data, nil
```

Comment:
this fixes the bug, thank you!

same comment as the others, id love to have a region prefix or something on here unless aws guarantees these ids are unique across regions.

that said, this fixes a bug, so im gonna go ahead and get this merged in